### PR TITLE
Updates to openstack terraform deploy

### PR DESCRIPTION
**Pull request by apearson-ibm**
Minor updates to the openstack terraform deployment.

@jwcroppe - please review and make sure in particular your OK with removing the network-id...I know older versions of Terraform required this, but i think it's safe to assume modern Terraform

### CODE REVIEW

**Comment by jwcroppe** [comment]
@apearson-ibm Changes look good - quick question though ... assume this also works if ssh login was already allowed (i.e., the script won't cause any harm)?

File: terraform/openstack/variables.tf [hunk 1]
```diff
@@ -64,7 +64,7 @@ variable "openstack_network_name" {
 }
 
 variable "openstack_ssh_key_file" {
-    description = "The path to the SSH key file."
+    description = "The path to the private SSH key file."
     default = "<path to SSH key file>"
```
Comment (jwcroppe) [inline terraform/openstack/variables.tf]: @apearson-ibm We should also put <path to SSH private key file> for completeness.

Comment (apearson-ibm) [reply terraform/openstack/variables.tf]: @jwcroppe Done

Regarding what happens if SSH login is already allowed - correct...it checks if it's allowed and if so, does nothing.  The check is to see whether there is a 'command="echo...' in the ssh authorized keys file...which is what cloud-init does.  It's possible somebody is using that for something else, but very unlikely.

File: terraform/openstack/bootstrap_icp_ce_master.sh [hunk 2]
```diff
@@ -41,6 +41,14 @@ ICP_ROOT_DIR="/opt/ibm-cloud-private-ce"
 IP=`/sbin/ifconfig eth0 | grep 'inet addr' | cut -d: -f2 | awk '{print $1}'`
 /bin/echo "${IP} $(hostname)" >> /etc/hosts
 
+# Make sure we can ssh as root w/ keys
+# This removes 'command=' when root login is disabled by cloud-init
+/bin/sed -ir 's#command="ech.*[^\]"##g' ~/.ssh/authorized_keys
```
Comment (paulczar) [inline terraform/openstack/bootstrap_icp_ce_master.sh]: should we really be using root login here?  we should use their cloud user's login `ubuntu` with sudo.    enabling remote root login is a big anti-pattern.

Comment (apearson-ibm) [reply terraform/openstack/bootstrap_icp_ce_master.sh]: @paulczar The ICP install documentation points to leveraging root across the nodes (https://www-03preprod.ibm.com/support/knowledgecenter/SSBS6K_2.1.0/installing/ssh_keys.html)

But your comment made me dig further - under Advanced Settings in the config.yaml there are `ansible_user` and `ansible_become` options which allow you to use users other than root.

I'll modify this PR to leverage those instead.  Thanks!

Comment (paulczar) [reply terraform/openstack/bootstrap_icp_ce_master.sh]: awesome!

**Comment by apearson-ibm** [comment]
I'm going to close this out, squash my commits and push up again to clean things up.

**Pull request closed without merging**